source/WindowScroller/utils/dimensions.js

```diff
@@ -16,12 +16,14 @@ type WindowScrollerProps = {
   serverWidth: number,
 };
 
-const isWindow = element => element === window;
+export const isWindow = (element: ?any) =>
+  element === window ||
+  (element && element.location && element.document && element.frameElement);
 
 const getBoundingBox = element => element.getBoundingClientRect();
 
 export function getDimensions(
-  scrollElement: ?Element,
+  scrollElement: ?any,
```

Comment:
We like to avoid `any` There are valid flow types for html dom elements. https://www.saltycrane.com/flow-type-cheat-sheet/latest/